GeoHealthCheck/config_main.py

```diff
@@ -47,6 +47,7 @@
 GHC_NOTIFICATIONS_EMAIL = ['you2@example.com']
 GHC_SITE_TITLE = 'GeoHealthCheck Demonstration'
 GHC_SITE_URL = 'http://host'
+GHC_METADATA_CACHE_SECS = 7200
```

Comment:
See #148 configs were even read at 3 locations. Now any module can just do 
```
from init import App
```
and use static functions like `App.get_app()` to get Flask App or `App.get_config()` to get app.config etc.